<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomRanges</class>
 <widget class="QDialog" name="CustomRanges">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>342</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>441</width>
    <height>342</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Customize color ranges</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Color ranges:</string>
         </property>
         <property name="buddy">
          <cstring>listRanges</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listRanges"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
         <item>
          <widget class="QToolButton" name="cmdAdd">
           <property name="text">
            <string>&amp;Add</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonFollowStyle</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="cmdDelete">
           <property name="text">
            <string>&amp;Delete</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonFollowStyle</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbMoreOptions">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&amp;More </string>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="boxRangeEditor">
       <property name="whatsThis">
        <string>The average shade of the image's recolored portions.</string>
       </property>
       <property name="title">
        <string>Colors</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="whatsThis">
           <string>The average shade of an image's recolored portions.</string>
          </property>
          <property name="text">
           <string>A&amp;verage:</string>
          </property>
          <property name="buddy">
           <cstring>leAvg</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="leAvg">
            <property name="whatsThis">
             <string>The average shade of an image's recolored portions.</string>
            </property>
            <property name="text">
             <string>#888888</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbAvg">
            <property name="whatsThis">
             <string>The average shade of an image's recolored portions.</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="whatsThis">
           <string>The maximum highlight shade of an image's recolored portions.</string>
          </property>
          <property name="text">
           <string>&amp;Highlight:</string>
          </property>
          <property name="buddy">
           <cstring>leMax</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="leMax">
            <property name="whatsThis">
             <string>The maximum highlight shade of an image's recolored portions.</string>
            </property>
            <property name="text">
             <string>#ffffff</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbMax">
            <property name="whatsThis">
             <string>The maximum highlight shade of an image's recolored portions.</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="whatsThis">
           <string>The minimum shadow shade of an image's recolored portions.</string>
          </property>
          <property name="text">
           <string>&amp;Shadow:</string>
          </property>
          <property name="buddy">
           <cstring>leMin</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="leMin">
            <property name="whatsThis">
             <string>The minimum shadow shade of an image's recolored portions.</string>
            </property>
            <property name="text">
             <string>#000000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbMin">
            <property name="whatsThis">
             <string>The minimum shadow shade of an image's recolored portions.</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="cmdAddFromList">
            <property name="text">
             <string>From &amp;List</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdWml">
            <property name="whatsThis">
             <string>Generates a sample Wesnoth Markup Language snippet containing a single color range definition, which you can save to a file or clipboard for later inclusion in your own project.</string>
            </property>
            <property name="text">
             <string>Generate &amp;WML</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_Rename">
   <property name="text">
    <string>&amp;Rename</string>
   </property>
  </action>
  <action name="action_Duplicate">
   <property name="text">
    <string>&amp;Duplicate</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>listRanges</tabstop>
  <tabstop>cmdAdd</tabstop>
  <tabstop>cmdDelete</tabstop>
  <tabstop>tbMoreOptions</tabstop>
  <tabstop>leAvg</tabstop>
  <tabstop>tbAvg</tabstop>
  <tabstop>leMax</tabstop>
  <tabstop>tbMax</tabstop>
  <tabstop>leMin</tabstop>
  <tabstop>tbMin</tabstop>
  <tabstop>cmdWml</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CustomRanges</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CustomRanges</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
